packages/gatsby/cache-dir/static-entry.js

```diff
@@ -62,51 +51,16 @@ const getPageDataUrl = pagePath => {
   return `${__PATH_PREFIX__}/${pageDataPath}`
 }
 
-const getStaticQueryUrl = hash =>
-  `${__PATH_PREFIX__}/page-data/sq/d/${hash}.json`
-
-const getPageData = pagePath => {
-  const pageDataPath = getPageDataPath(pagePath)
-  const absolutePageDataPath = join(process.cwd(), `public`, pageDataPath)
-  const pageDataRaw = fs.readFileSync(absolutePageDataPath)
-
-  try {
-    return JSON.parse(pageDataRaw.toString())
-  } catch (err) {
-    return null
-  }
-}
-
-const appDataPath = join(`page-data`, `app-data.json`)
-
-const getAppDataUrl = memoize(() => {
-  let appData
+const getStaticQueryPath = hash => join(`page-data`, `sq`, `d`, `${hash}.json`)
 
-  try {
-    const absoluteAppDataPath = join(process.cwd(), `public`, appDataPath)
-    const appDataRaw = fs.readFileSync(absoluteAppDataPath)
-    appData = JSON.parse(appDataRaw.toString())
-
-    if (!appData) {
-      return null
-    }
-  } catch (err) {
-    return null
-  }
+const getStaticQueryUrl = hash =>
+  `${__PATH_PREFIX__}/${getStaticQueryPath(hash)}`
 
-  return `${__PATH_PREFIX__}/${appDataPath}`
-})
+const getAppDataUrl = () =>
+  `${__PATH_PREFIX__}/${join(`page-data`, `app-data.json`)}`
 
-const loadPageDataSync = pagePath => {
-  const pageDataPath = getPageDataPath(pagePath)
-  const pageDataFile = join(process.cwd(), `public`, pageDataPath)
-  try {
-    const pageDataJson = fs.readFileSync(pageDataFile)
-    return JSON.parse(pageDataJson)
-  } catch (error) {
-    // not an error if file is not found. There's just no page data
-    return null
-  }
+function loadPageDataSync() {
+  throw new Error(`"loadPageDataSync" is no longer available`)
```

Comment:
Breaking change no #2

This might not need to be done, but only plugin that was making use of this was `guess-js` one

Reply:
This might be changed to "temporarairly unavailble, but using this will cause all pages to be rebuild all the time" when we wrap `fs` usage.

That would be safer given, that we didn't have deprecation for it, so we don't need to break it completely (so it continue to work but usage of it would disallow using partial html writes)

Reply:
https://www.npmjs.com/package/gatsby-plugin-guess-js gets almost no downloads so I think it's fine saying adios.